app/views/role_in_the_business.scala.html

```diff
@@ -0,0 +1,71 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.data.Form
+@import play.api.mvc.Request
+@import play.api.i18n.Messages
+
+@import config.FrontendAppConfig
+
+@import views.html.layouts.layout
+@import views.html.components.h1
+@import views.html.components.inputRadio
+@import views.html.components.errorSummary
+@import controllers.registration.applicant.{routes => applicantRoutes}
+
+
+@this(
+        layout: layout,
+        govukButton: GovukButton,
+        formWithCSRF: FormWithCSRF,
+        h1: h1,
+        inputRadio: inputRadio,
+        errorSummary: errorSummary
+)
+
+@(form: Form[RoleInTheBusiness])(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@layout(pageTitle = Some(title(form, messages("pages.RoleInTheBusiness.heading")))){
+
+    @errorSummary(form.errors)
+
+   @h1("pages.RoleInTheBusiness.heading")
```

Comment:
Remove this as there's only one input on the page & no content between the heading and input, so the radio legend should be the heading